### DISCUSSION TITLE
Export comments ?

### DISCUSSION THREAD

**Discussion by 0ldb34r** [iSoron/uhabits#1978] [Help & FAQ]
Hello,
I'm tracking a mood score on the app and had noticed that there was the option to add a comment. So I regularly fill in the comment field to help me process the data later, when exporting. However, I've just realized that when I export to CSV, I can't find the comments I've entered. Is there a way of exporting them?

**Comment by studgeek** [comment]
I'm not seeing them either. I went through all of the CSVs.

**Comment by studgeek** [comment]
I believe this is a duplicate of #1567. So please comment/vote there also.
